airflow/providers/google/marketing_platform/operators/display_video.py

```diff
@@ -221,10 +223,10 @@ def __init__(
         self.delegate_to = delegate_to
 
     def _resolve_file_name(self, name: str) -> str:
-        csv = ".csv"
+        csv_file = ".csv"
```

Comment:
@turbaszek pls, check it. I had to change it, becasue I imported CSV module.

Reply:
I think we can simplify it, please take a look at my comments ;)